### PR TITLE
Feature/chart data connection

### PR DESCRIPTION
**Pull request by bitTal**
Connects the data to the chart depending on the month specified & changes line chart to bar chart.

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -179,6 +181,11 @@ class App extends React.Component {
     this.setState(newState);
   }
 
+  getMonth() {
+    let date = this.state.timelineDate;
+    return date.split('-')[1];
```
Comment (dhakelila) [inline src/app.jsx]: moment has a method to get the month. As we are already loading moment here, you can use it.

File: src/app.jsx [hunk 2]
```diff
@@ -187,10 +189,14 @@ class App extends React.Component {
     this.setState(newState);
   }
 
+  getMonth() {
+    const date = moment(this.state.timelineDate);
+    return date.month()+1;
+  }
+
```
Comment (dhakelila) [inline src/app.jsx:253]: I think this method should live into chart component. 
Unless you think with need it also for another component.

**Comment by dhakelila** [comment]
It looks very good to me! 
But lets see what @simaob thinks about how it looks.

**Comment by simaob** [comment]
Yes, I like it =) works better than the line chart.

**Pull request merged**